### DISCUSSION TITLE
Comfort model indoor

### DISCUSSION THREAD

**Discussion by uroh** [dolezsa/thermal_comfort#263] [Ideas]
Hello everyone, is it possible to also integrate the Fanger PMV model, the Adaptive Model, and the ASHRAE Standard 55-2020 comfort index into the component (see https://comfort.cbe.berkeley.edu/)? Currently, I calculate it through the libraries provided by Berkeley (node-red). I have never seen these indoor comfort indicators mentioned in discussions, however, I find them particularly useful for those who have an HVAC. Thank you.

**Comment by rautesamtr** [comment]
This sounds pretty interesting, I currently just don't have the time to look further into this but will definitly give it a try once i get some more free time. #149
